engines/bops_consultees/app/controllers/bops_consultees/planning_applications_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module BopsConsultees
+  class PlanningApplicationsController < ApplicationController
+    before_action :authenticate_with_sgid!
+    before_action :find_planning_application, only: :show
+
+    def show
+      respond_to do |format|
+        format.html
+      end
+    end
+
+    private
+
+    def find_planning_application
```

Comment:
would it be more consistent to call this `set_planning_application`? minor i know.